lib/ansible/modules/windows/win_certificate_store.py

```diff
@@ -38,7 +38,7 @@
     description:
     - The path to a certificate file.
     - This is required when I(state) is C(present) or C(exported).
-    - When I(state) is C(absent) and I(thumbprint) is not specified, the
+    - When I(state) is C(absent) and I(thumbprint) is not specified, the=
```

Comment:
Could you remove the stray `=` here?

Reply:
I think the stray `=` was probably a typo . . . or a cat on your keyboard.

Reply:
Dangit :Cat: